src/nvim/window.c

```diff
@@ -1044,19 +1044,21 @@ static void win_init_some(win_T *newp, win_T *oldp)
 }
 
 
-/*
- * Check if "win" is a pointer to an existing window.
- */
-int win_valid(win_T *win)
+/// Return true if `win` is a pointer to an existing window.
```

Comment:
Do we want to change variable names to backtick? Yes, there are a few precendents in newly written code, but `"variablename"` is much more common in the codebase, and IIRC most refactors of this kind have kept the "" form.